packages/rip2/build.sh

```diff
@@ -0,0 +1,32 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/MilesCranmer/rip2
+TERMUX_PKG_DESCRIPTION="A safe and ergonomic alternative to rm"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="Joshua Kahn @TomJo2000"
+TERMUX_PKG_VERSION="0.8.2"
+TERMUX_PKG_SRCURL="https://github.com/MilesCranmer/rip2/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz"
+TERMUX_PKG_SHA256=dab2b04bbec88f5789687acff1f85196f3253e871ed5430d6835e73eab924888
+TERMUX_PKG_BREAKS="rip"
+TERMUX_PKG_REPLACES="rip"
+TERMUX_PKG_BUILD_IN_SRC=true
```

Comment:
We need to make sure the old package is going to be removed from device, TERMUX_PKG_CONFLICT should do that, I am not sure if TERMUX_PKG_BREAKS implements this.

Reply:
Just a `TERMUX_PKG_REPLACES` should be enough.
That's what we did with `anacron` when that got retired last week.
https://github.com/termux/termux-packages/blob/13be95a87619afd8fd399a3386a578a772dbb38f/packages/cronie/build.sh#L16
- #21805